### ISSUE TITLE
Allow choosing number display format in a calculated row in a data cell

### ISSUE THREAD

**Issue by MisterE123** [mgreminger/EngineeringPaper.xyz#286]
![image](https://github.com/user-attachments/assets/b67aef65-bdaf-4a80-a163-7bf4d851e2a7)

As the image shows, the calculated column displays all 15 digits of precision. I would have liked to truncate that to 4 sig figs. Really, calculated rows should have all the display options (symbolic, sigfigs, SciNotation, etc) that calculation cells have.

**Comment by mgreminger** [comment]
Thanks for raising the issue. For efficiency, EP uses floating point operations for data table values rather than the arbitrary precision math used in other EP calculations. Because of this, the same logic for floating point precision cannot be applied to data table cells as is applied to other outputs in EP. However, it is agreed that some control over the output precision should be provided for data tables and likely will be eventually added.

**Comment by MisterE123** [comment]
Yes, it would be good to at least be able to control the number of sig figs.
